src/pytest_container/runtime.py

```diff
@@ -189,10 +304,34 @@ def get_container_health(self, container_id: str) -> ContainerHealth:
         return ContainerHealth.UNHEALTHY
 
 
+def _get_docker_version(version_stdout: str) -> Version:
+    matches = re.match(
+        r"docker version (?P<major>\d+)\.(?P<minor>\d+)(\.(?P<patch>\d+)(\S+)?)?,"
```

Comment:
good, as docker version string is not that simple, e.g. `Docker version 20.10.6-ce, build 8728dd246c3a`